packages/vscode-extension/src/startup/StatusBars.ts

```diff
@@ -0,0 +1,135 @@
+import * as vscode from 'vscode';
```

Comment:
I don't have any smart suggestion, but maybe we should rethink how this file is called since the current name is not very descriptive about its usage.

Reply:
Renamed to `StatusBarItemsFunctions.ts`